docs/features.md

```diff
@@ -1,58 +1,266 @@
+
+Provided by: `hls-tactics-plugin`
+
+Provides a variety of code actions for interactive code development, see https://haskellwingman.dev/ for more details.
+
+![Wingman Demo](https://user-images.githubusercontent.com/307223/92657198-3d4be400-f2a9-11ea-8ad3-f541c8eea891.gif)
+
+## Code lenses
+
+### Add type signature
+Provided by: `ghcide`
+
+Shows the type signature for bindings without type signatures, and adds it with a click.
+
+### Evaluation code snippets in comments
+Status: Not supported on GHC 9.2
+
+Provided by: `hls-eval-plugin`
+
+Evaluates code blocks in comments with a click. [Tutorial](https://github.com/haskell/haskell-language-server/blob/master/plugins/hls-eval-plugin/README.md).
+
+![Eval Demo](https://raw.githubusercontent.com/haskell/haskell-language-server/master/plugins/hls-eval-plugin/demo.gif)
+
+### Make import lists fully explicit
+Provided by: `hls-explicit-imports-plugin`
+
+Shows fully explicit import lists and rewrites them with a click (same as the code action).
+
+![Imports code lens Demo](https://imgur.com/pX9kvY4.gif)
+
+### Refine import
+Provided by: `hls-refine-imports-plugin`
+
+Shows refined imports and applies them with a click (same as the code action).
+
+### Fix module names
+Provided by: `hls-module-name-plugin`
+
+Shows module name matching file path, and applies it with a click.
+
+![Module Name Demo](https://user-images.githubusercontent.com/54035/110860755-78ad8680-82bd-11eb-9845-9ea4b1cc1f76.gif)
+
+## Missing features
+
+The following features are supported by the LSP specification but not implemented in HLS.
+Contributions welcome!
+
+| Feature                | Status                                                                                   | [LSP method](./what-is-hls.md#lsp-terminology)      |
+|------------------------|------------------------------------------------------------------------------------------|-----------------------------------------------------|
+| Signature help         | Unimplemented                                                                            | `textDocument/signatureHelp`                        |
+| Jump to declaration    | Unclear if useful                                                                        | `textDocument/declaration`                          |
+| Jump to implementation | Unclear if useful                                                                        | `textDocument/implementation`                       |
+| Renaming               | [Parital implementation](https://github.com/haskell/haskell-language-server/issues/2193) | `textDocument/rename`, `textDocument/prepareRename` |
+| Folding                | Unimplemented                                                                            | `textDocument/foldingRange`                         |
+| Selection range        | Unimplemented                                                                            | `textDocument/selectionRange`                       |
```

Comment:
does it worth mention the wip pr #2565?

Reply:
Eh, I'd wait until it's merged. It might never be, who knows!